youtube_dl_gui/locale/cs_CZ/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,589 @@
+# Youtube-dlG localization file.
+# FIRST AUTHOR: Sotiris Papadopoulos <ytubedlg@gmail.com>, 2015.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: youtube-dlg 0.4\n"
+"POT-Creation-Date: 2017-06-15 17:13+EEST\n"
+"PO-Revision-Date: 2017-10-08 00:30+0200\n"
+"Last-Translator: Pavel Řehák <pavel-rehak@email.cz>\n"
+"Language-Team: \n"
+"Language: cs_CZ\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: pygettext.py 1.5\n"
+"X-Generator: Poedit 1.8.7.1\n"
+
+#: youtube_dl_gui/formats.py:9 youtube_dl_gui/formats.py:108
+msgid "ID"
+msgstr "ID"
+
+#: youtube_dl_gui/formats.py:10 youtube_dl_gui/formats.py:109
+#: youtube_dl_gui/mainframe.py:140
+msgid "Title"
+msgstr "Název"
+
+#: youtube_dl_gui/formats.py:11 youtube_dl_gui/formats.py:110
+msgid "Title + ID"
+msgstr "Název + ID"
+
+#: youtube_dl_gui/formats.py:12 youtube_dl_gui/formats.py:111
+msgid "Title + Quality"
+msgstr "Název + Kvalita"
+
+#: youtube_dl_gui/formats.py:13 youtube_dl_gui/formats.py:112
+msgid "Title + ID + Quality"
+msgstr "Název + ID + Kvalita"
+
+#: youtube_dl_gui/formats.py:14 youtube_dl_gui/formats.py:113
+msgid "Custom"
+msgstr "Vlastní"
+
+#: youtube_dl_gui/formats.py:19 youtube_dl_gui/formats.py:118
+#: youtube_dl_gui/mainframe.py:503 youtube_dl_gui/mainframe.py:506
+msgid "default"
+msgstr "výchozí"
+
+#: youtube_dl_gui/mainframe.py:97
+msgid "Enter URLs below"
+msgstr "Níže vložte URL adresy"
+
+#: youtube_dl_gui/mainframe.py:98
+msgid "Update"
+msgstr "Aktualizovat"
+
+#: youtube_dl_gui/mainframe.py:99 youtube_dl_gui/optionsframe.py:41
+msgid "Options"
+msgstr "Volby"
+
+#: youtube_dl_gui/mainframe.py:100 youtube_dl_gui/optionsframe.py:589
+msgid "Stop"
+msgstr "Zastavit"
+
+#: youtube_dl_gui/mainframe.py:101
+msgid "Info"
+msgstr "Informace"
+
+#: youtube_dl_gui/mainframe.py:102
+msgid "Welcome"
+msgstr "Vítejte"
+
+#: youtube_dl_gui/mainframe.py:103
+msgid "Warning"
+msgstr "Upozornění"
+
+#: youtube_dl_gui/mainframe.py:105
+msgid "Add"
+msgstr "Přidat"
+
+#: youtube_dl_gui/mainframe.py:106
+msgid "Download list"
+msgstr "Seznam stahovaných"
```

Comment:
This *msgstr* translates to "List downloaded". Shouldn't this be "Seznam stahování" or am i wrong?

Reply:
"Seznam stahovaných" means "List of some things, which are downloading". But you are right, "Seznam stahování" is common in a software.